browser/ui/webui/settings/brave_privacy_handler.cc

```diff
@@ -9,20 +9,37 @@
 
 #include "base/bind.h"
 #include "base/values.h"
+#include "brave/common/pref_names.h"
 #include "brave/components/p3a/pref_names.h"
 #include "chrome/browser/browser_process.h"
 #include "chrome/browser/profiles/profile.h"
 #include "chrome/common/pref_names.h"
 #include "components/gcm_driver/gcm_buildflags.h"
 #include "components/prefs/pref_service.h"
 #include "content/public/browser/web_ui.h"
-#include "content/public/common/webrtc_ip_handling_policy.h"
 #include "content/public/browser/web_ui_data_source.h"
+#include "content/public/common/webrtc_ip_handling_policy.h"
 
 #if !BUILDFLAG(USE_GCM_FROM_PLATFORM)
 #include "brave/browser/gcm_driver/brave_gcm_channel_status.h"
 #endif
 
+BravePrivacyHandler::BravePrivacyHandler() {
+  local_state_change_registrar_.Init(g_browser_process->local_state());
+  local_state_change_registrar_.Add(
```

Comment:
p3a? Should be a separate PR?

Reply:
I can definitely do it in another PR 😄 (following this). It's something I had missed originally. Will update to remove this

Reply:
Updated to remove the P3A pieces; can do in a follow up